### PR TITLE
UI changes to select target IoT SKU

### PR DESCRIPTION
**Pull request by hathind-ms**
# Type of change? 
Changes to add UI for selecting target IoTHub Sku in order to update RateLimits (Hub throttling settings)

- [ ] Bug fix
- [x] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 

...

**Checklist:**

- [x] All tests passed
- [x] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: src/components/pages/simulation/views/simulationForm.scss [hunk 1]
```diff
@@ -86,6 +86,42 @@
   }
 }
 
+.hub-units-box {
+  display: flex;
+
+  .warning-desc {
```
Comment (spryor) [inline src/components/pages/simulation/views/simulationForm.scss]: nit: should be one line

File: src/components/pages/simulation/views/simulationForm.scss [hunk 2]
```diff
@@ -86,6 +86,42 @@
...
+    @include rem-fallback(margin-left, 6px);
+  }
+
+  .warning-label {
+      font-weight: bold;
```
Comment (spryor) [inline src/components/pages/simulation/views/simulationForm.scss]: two space tabs

File: src/services/models/simulationModels.js [hunk 3]
```diff
@@ -146,3 +148,31 @@ const toIoTHubs = (iotHubs = []) =>
   iotHubs.map(({ connectionString }) => {
     return { ConnectionString: connectionString };
   });
+
+// Map to rateLimits in simulation request model
+const toRateLimits = (iotHubSku = 'S2', iotHubUnits = 1) => {
+  var rateLimits = {};
```
Comment (spryor) [inline src/services/models/simulationModels.js]: I would use `let` instead of `var`

File: src/services/simulationService.js [hunk 4]
```diff
@@ -54,6 +54,8 @@ export class SimulationService {
 
   /** Creates a new simulation */
   static createSimulation(model) {
+    var mod = toSimulationRequestModel(model);
```
Comment (spryor) [inline src/services/simulationService.js]: Doesn't look like this needs to be here. If it does, use `const`.

Comment (hathind-ms) [reply src/services/simulationService.js]: removed, not needed

File: src/components/pages/simulation/views/simulationForm.scss (hunk 1 above)
Comment (hathind-ms) [reply src/components/pages/simulation/views/simulationForm.scss]: fixed

File: src/components/pages/simulation/views/simulationForm.scss (hunk 2 above)
Comment (hathind-ms) [reply src/components/pages/simulation/views/simulationForm.scss]: fixed

File: src/components/pages/simulation/views/simulationForm.js [hunk 5]
```diff
@@ -422,6 +449,41 @@ class SimulationForm extends LinkedComponent {
...
+            </div>
+            <FormLabel className="hub-units-desc">{t('simulation.form.targetHub.sku.unitsLabel')}</FormLabel>
+            <FormGroup className="hub-units-box">
+              <FormControl
+                className="short"
+                type="text"
+                link={this.iotHubUnits}
+                max={10} />
+              <FormLabel className="warning-label">{t('simulation.form.targetHub.sku.warningLabel')}</FormLabel>
+              <FormLabel className="warning-desc">{t('simulation.form.targetHub.sku.warningMessage',{ messageThrottlingLimit })}</FormLabel>
```
Comment (maple10001) [inline src/components/pages/simulation/views/simulationForm.js]: nit: `{t('simulation.form.targetHub.sku.warningMessage', { messageThrottlingLimit })}`

**Pull request merged**